internal/handler/msg_updateuser.go

```diff
@@ -29,6 +38,201 @@ func (h *Handler) MsgUpdateUser(ctx context.Context, msg *wire.OpMsg) (*wire.OpM
 		return nil, lazyerrors.Error(err)
 	}
 
-	// TODO https://github.com/FerretDB/FerretDB/issues/1496
-	return nil, common.Unimplemented(document, document.Command())
+	dbName, err := common.GetRequiredParam[string](document, "$db")
+	if err != nil {
+		return nil, lazyerrors.Error(err)
```

Comment:
Minor comment for here and below: I think errors returned after `GetRequiredParam` don't have to be `lazyerrors`, because `GetRequiredParam` returns a user-friendly error that we just need to show as is.